std/experimental/logger/core.d

```diff
@@ -1763,7 +1763,7 @@ private @property Logger stdThreadLocalLogImpl() @trusted
 {
     import std.conv : emplace;
 
-    static ubyte[__traits(classInstanceSize, StdForwardLogger)] _buffer;
+    static /*align(StdForwardLogger.alignof)*/ void[__traits(classInstanceSize, StdForwardLogger)] _buffer;
```

Comment:
There's `divideRoundUp` in std.experimental.allocator, perhaps it should be moved up.